.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,8 +30,11 @@
 - [ ] Changes were tested in the Kubernetes cluster manually.
 
 ## Changelog entries
+<!---
+  for the list of allowed sections, see https://github.com/deckhouse/virtualization/blob/main/.github/actions/milestone-changelog/action.yml#L28 
```

Comment:
We have CONTRIBUTING.md, I think its a better source for understanding what to include in the change entry.